release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1711,7 +1711,7 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-13-rhv-deprecations"]
 ==== Red Hat Virtualization (RHV) as a host platform for {product-title} will be deprecated
```

Comment:
it says "will be" here, is that consistent?

Reply:
Good catch. Updated!